src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -156,7 +156,7 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
             SUBSCRIPTION_TAB_INDEX)
       }
       subscriptionPanel.update(data.isCurrentUserPro)
-    } else {
+    } else if (SUBSCRIPTION_TAB_INDEX < tabbedPane.tabCount) {
```

Comment:
unrel to the sourcegraph actions issue - minor fix (`IndexOutOfBoundsException` appeared once from this place)